addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -162,6 +162,19 @@ def in_eu(partner):
                 if tax.amount == 0.0:
                     tax_map[tax] = tax_map.get(tax, 0.0) + line.price_subtotal
 
+        # Constraints within the edi make local rounding on price included taxes a problem.
+        # To solve this there is a <Arrotondamento> or 'rounding' field, such that:
+        #   taxable base = sum(taxable base for each unit) + Arrotondamento
+        tax_details = self._prepare_edi_tax_details()
+        for _tax_name, tax_dict in tax_details['tax_details'].items():
+            base_amount = tax_dict['base_amount']
+            tax_amount = tax_dict['tax_amount']
+            tax_rate = tax_dict['tax'].amount
+
+            if float_compare(base_amount, float_round(tax_amount * 100 / tax_rate, 2), 2) and tax_dict['tax'].price_include:
```

Comment:
@dbkosky Should not we check if the type is percentage as well?  (not sure if they use other types of taxes a lot, but better to avoid it anyways)  and should not we set the values in tax_dict anyways?  You could also do if price_include or not percentage: continue.